ApiReview/API.Polly.Core/NoDocs/Polly.Timeout/TimeoutStrategyOptions.cs

```diff
@@ -0,0 +1,17 @@
+// Assembly 'Polly.Core'
+
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+
+namespace Polly.Timeout;
+
+public class TimeoutStrategyOptions : ResilienceStrategyOptions
+{
+    [Range(typeof(TimeSpan), "00:00:01", "1.00:00:00")]
+    public TimeSpan Timeout { get; set; }
```

Comment:
is this a good name?

```suggestion
    public TimeSpan DefaultTimeout { get; set; }
```

Reply:
I am 50/50 on both.